src/OpenLoco/src/MessageManager.cpp

```diff
@@ -24,7 +24,7 @@ namespace OpenLoco::MessageManager
     static void remove(const MessageId id);
 
     static auto& rawMessages() { return getGameState().messages; }
-    static auto& numMessages() { return getGameState().numMessages; }
+    uint16_t& getNumMessages() { return getGameState().numMessages; }
```

Comment:
This should be the only file that will ever modify the message count (and even then there will only be about 3 modifiers add/remove/reset). All other users will just need this as const.